mobile/test/java/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -266,9 +263,6 @@ class EnvoyConfigurationTest {
     assertThat(resolvedTemplate).contains("address: \"1.2.3.4\"");
     assertThat(resolvedTemplate).contains("port_value: 88");
 
-    // enableGro = true
```

Comment:
do we want to leave this in, given we still want to make sure we explicitly set for now?

Reply:
good point, re-added

Reply:
actually, since we made the flag default true now, this is no longer an applicable check (and causes the test to fail)